AirportsLatLongConverter.py

```diff
@@ -0,0 +1,20 @@
+import airportsdata
+
+#airports = airportsdata.load('IATA')
+
+# Should we add a name to abbreviation converter?
+# How will users input their airport?
+
+# Get lat and lon
+def get_lat_and_lon(input_airport):

```

Comment:
unit tests for this function?  what values are expected given a valid airport like `KOKC` or `KLAX`?  what behavior is expected if we give invalid airport names like `DNE` or `None`?

Reply:
It runs to error in all cases that aren't in IATA format ('DFW', 'OKC', etc). I'll make a note to return a string statement if the format is incorrect.

Reply:
Sir, I'm going to add this as an issue to work on to return a string.